src/main/java/com/solvd/pages/HomePage.java

```diff
@@ -16,13 +17,17 @@ public class HomePage {
     private WebElement searchButton;
 
     public HomePage(WebDriver driver){
-        this.driver = driver;
-        PageFactory.initElements(driver, this);
+        super(driver);
+        driver.get(ConfigReader.getProperty("url"));
```

Comment:
i think that driver.get(ConfigReader.getProperty("url")) should be in separate method like goToPage(), because if one day you will have a method that after clicking some button will return newHomePage(driver) then instead of being redirected from that button to home page you will open home page directly.

Reply:
already fixed it according to your comments on the previous pull request